main/src/ca/uwaterloo/flix/language/phase/Verifier.scala

```diff
@@ -616,6 +616,49 @@ object Verifier {
     }
   }
 
+  /**
+   * Returns a corresponding Java class for the given monotype.
+   */
+  def getJavaType(tpe: MonoType, loc: SourceLocation): Class[_] = {
```

Comment:
Jonathan says this is fine

Reply:
Ok thank you. It isn't complete for now as we want to have a working version.